site-manage/pageorder/tool/src/webapp/content/templates/PageList.html

```diff
@@ -36,7 +36,7 @@ <h1 rsf:id="msg=page_title">page_title</h1>
 									<div class="item_edit_box" style="display: none;">
 										<div>
 											<label rsf:id="page-name-label"for="page-name-input">Title</label>
-											<input rsf:id="page-name-input" class="new_title" id="page-name-input"></input>
+											<input rsf:id="page-name-input" class="new_title" maxlength="20" id="page-name-input"></input>
```

Comment:
Is this always 20?

Reply:
It was the proposed value by the Jira creator, and matches the max value on that other PR I made. It comes from this constant:

`private final static int MAX_TOOL_TITLE_LENGTH = 20;`

So I'd say yes...

Reply:
In that case you should supply it, not hardcode it, like SiteAction does, imo.

Reply:
Ok, seems fair. It's still a different tool, so I can't use the same variable, which is not ideal.